test/no_groups_submission.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
This file is actually identical to the previous version of `test_submission.json`.